src/Enterspeed.Source.UmbracoCms.V10/Composers/EnterspeedComposer.cs

```diff
@@ -57,6 +58,7 @@ public void Compose(IUmbracoBuilder builder)
             builder.Services.AddTransient<IUrlFactory, UrlFactory>();
             builder.Services.AddTransient<IEnterspeedJobFactory, EnterspeedJobFactory>();
             builder.Services.AddTransient<IEnterspeedJobsHandlingService, EnterspeedJobsHandlingService>();
+            builder.Services.AddTransient<IEnterspeedPropertyValueValidator, EnterspeedPropertyValueValidator>();
```

Comment:
Why not singleton, as I see it the validator doesn't have any state?

Reply:
You are right